### ISSUE TITLE
Support combining several DittoProtocol commands in a single aggregating one

### ISSUE THREAD

**Issue by thjaeckle** [eclipse/ditto#289]
In #288 an alternative solution for doing multiple "modify" operations by only issuing a single DittoProtocol command was suggested:

Adding a DittoProtocol command which contains several other DittoProtocol commands (as simple Json array in its JSON form).

As this would be simpler than #288 (adding a "PATCH" command), we track it separately.

**Comment by avgustinmm** [comment]
It seems like something that we could use. In our case we get batch of change events from the gateways that we pass to ditto (through hono) using a hono message per command. A batch command for ditto means that we could send a single message per multiple commands that should optimize the hono usage (e.h. tragic, latency ..). I, in theory, could allow ditto service to use the underlying database in more efficient way.

**Comment by yufei-cai** [comment]
Is this relevant after #288?

**Comment by thjaeckle** [comment]
It could still be, but with even lower priority when #288 is in place.
